### ISSUE TITLE
Grafana 2.0.2 Units? bug (OpenTSDB series)

### ISSUE THREAD

**Issue by jmaasing** [grafana/grafana#1951]
I just installed Grafana 2.0.2 and imported a dashboard from version 1.9.1.
The graphs have strange values for the units and the series-labels seems to be wrong.
In grafana 1.9 this is what it looks like (which is correct)

![grafana19](https://cloud.githubusercontent.com/assets/871057/7492208/b616029c-f3f1-11e4-98e3-52097cf2b58e.png)

The imported dashboard in Grafana 2.0.2 shows these values:

![grafana202](https://cloud.githubusercontent.com/assets/871057/7492214/ccdfee66-f3f1-11e4-91c9-e17084b527d3.png)

These are heap memory usage numbers, the 'spiky' curve is actually around 800 MB usage, the flatter lines are max usage and should be around 2 GB. In grafana 2.0.2 however the units seems strange. I've tried to change in the GUI from 'short' to bytes et c. but they are still very off. 

The graph is composed of three queries, e.g. 'heap.used' 'heap.max'.., each query gets two series since we are monitoring two hosts. So the queries are of the form 'heap.used host=*', 'heap.max host=*'

I have tried to graph one query at a time and also using host=X to get just one series but I end up with no change in the graphs, i.e. they show the same error as the 'composite' graph.

**Comment by torkelo** [comment]
Can you inspect the data returned by OpenTSDB? (open chrome dev tools and check network tab). No change in how data is processed in Grafana for v2.0 nor in OpenTSDB request/response handling. So very strange. If you get the same values when using "none" unit, it shows the values as they come from OpenTSDB.

**Comment by jmaasing** [comment]
I found the problem and I don't think it's a bug as such, more unexpected behaviour. 
In the GUI I had an empty field for downsampling - which suggest Interval (empty = auto). I interpreted the empty field as downsampling was disabled - but that isn't the case. When I looked at the query what grafana sends is "downsample":"30s-sum". 

This has a weird effect, if I look at data that is in the time span "Last 5m" the graph looks OK. However, the larger time span I select in the GUI the larger the values will be since the downsampling will actually sum all the 'skipped' values. So if I select "Last 2d" you get the weird effect I reported.

If I Disable downsampling in the GUI the graph looks correct since now it mimics the behaviour of Grafan1.

So not a bug, but a suggestion would be to either use 'avg' as the default value for downsampling aggregator or preferably disable downsampling when importing a dashboard.
